src/master-playlist-controller.js

```diff
@@ -1287,8 +1287,21 @@ export class MasterPlaylistController extends videojs.EventTarget {
       return;
     }
 
-    const suggestedPresentationDelay = this.masterPlaylistLoader_.master.suggestedPresentationDelay;
-    const mainSeekable = Vhs.Playlist.seekable(media, expired, suggestedPresentationDelay);
+    let delay;
+
+    if (this.masterPlaylistLoader_.master.hasOwnProperty('suggestedPresentationDelay')) {
+      delay = this.masterPlaylistLoader_.master.suggestedPresentationDelay;
+    } else if (media.serverControl && media.serverControl['PART-HOLD-BACK'] && media.partTargetDuration) {
+      delay = media.serverControl['PART-HOLD-BACK'] * media.partTargetDuration;
+    } else if (media.serverControl && media.serverControl['HOLD-BACK'] && media.targetDuration) {
+      delay = media.serverControl['HOLD-BACK'] * media.targetDuration;
+    } else if (media.partTargetDuration) {
+      delay = media.partTargetDuration * 3;
+    } else if (media.targetDuration) {
+      delay = media.targetDuration * 3;
```

Comment:
These are the default values for `HOLD-BACK` as per the spec, so even if a playlist doesn't have `HOLD-BACK` we can still set the default `HOLD-BACK`